docs/integrations.md

```diff
@@ -31,3 +31,15 @@ __Link:__ [https://github.com/adfinis-sygroup/aem-docker/tree/master](https://gi
 __Description:__  "Download and unarchive the latest kompose release asset for your OS"
 
 __Link:__ [https://github.com/chouseknecht/kompose-install-role](https://github.com/chouseknecht/kompose-install-role)
+
+### Fabric8 Maven Plugin by Red Hat
+
+__Description:__  "To bring docker-compose based JAVA application to Kubernetes/Openshift Cluster using Maven Plugin"
```

Comment:
Use 'Java' instead of 'JAVA'. It is a [registered trademark](https://www.oracle.com/legal/trademarks.html).